### DISCUSSION TITLE
How to use Ethereum Provider in Vanilla Js

### DISCUSSION THREAD

**Discussion by orangeqq** [WalletConnect/walletconnect-monorepo#2939] [Developer Support]
It seems that the [Ethereum Provider](https://docs.walletconnect.com/2.0/web/providers/ethereum) module does not have a version available on a CDN with a bundle.js file, similar to the [WalletConnectModal](https://docs.walletconnect.com/2.0/web/walletConnectModal/modal/installation#).

Is there any way to use it in a simple html just like the following code?
```javascript
<script type="module">
    import { EthereumProvider } from 'https://unpkg.com/@walletconnect/ethereum-provider';
</script>
``` 
Currently, it shows an error in the browser console:
```
SyntaxError: Importing binding name 'EthereumProvider' is not found.
```

**Comment by crypblizz8** [comment]
@ganchoradkov 👀
